index.html

```diff
@@ -191,7 +191,7 @@ <h1 class="heading"><i class="fas fa-cloud"></i> sButton CDN</h1>
           <div class="footer-links">
             <a href="https://github.com/sButtons/sbuttons" target="_blank" title="View on Github"> <i
                 class="fab fa-github"></i> GitHub</a>
-            <a href="https://github.com/sButtons/sbuttons" target="_blank" title="View on Github"> <i
+            <a href="https://sbuttons.github.io/sbuttons/examples.html" target="_blank" title="View on Github"> <i
```

Comment:
@pratyushakatti13 Just one thing, Can you remove the target attribute.

Reply:
Yeah sure. It should be removed for only 'Examples' link or other 2 Links in Footer?

Reply:
Just for the `Examples`

Reply:
Its done!